src/main/java/edu/gonzaga/EpidemicCard.java

```diff
@@ -2,7 +2,56 @@
 
 public class EpidemicCard extends Card {
     
-    public void triggerEpidemic(){
+    /**
+     * Constructor for the EpidemicCard class
+     * @return an EpidemicCard object
+     * 
+     * @Author Tony
+     */
+    public void triggerEmpidemic(Gameboard board) {
+        // increase the infection rate
+        increaseInfectionRate(board);
 
+        // infect
+        infect(board);
+
+        // intensify
+        intensify(board);
+    }
+
+    /**
+     * Infects cities
+     * 
+     * @param board the gameboard
+     * @return void
+     * @Author Tony
+     */
+    private void infect(Gameboard board) {
+        // infect cities
+        board.takeInfectionTurn();
```

Comment:
The board shouldn't take an infection turn, this should take the card from the bottom of the infection deck, add 3 cubes, discard that card, and then move onto the intensify() function